src/auth-service/models/User.js

```diff
@@ -195,11 +195,17 @@ const UserSchema = new Schema(
     category: {
       type: String,
     },
-    notifications: {
-      email: { type: Boolean, default: false },
-      push: { type: Boolean, default: false },
-      text: { type: Boolean, default: false },
-      phone: { type: Boolean, default: false },
+    mobile_notifications: {
```

Comment:
this is a good direction @BenjaminSsempala , love it! :)

Reply:
Hi @BenjaminSsempala ....on a second thought, we might want to rethink this and just maintain one notifications field.......since it already had "phone" in it, we might not need to create a new field for mobile.

Happy to hear your additional thoughts on the same.